src/src_python/antares_xpansion/xpansionConfig.py

```diff
@@ -25,7 +40,6 @@ def __init__(self):
 
```

Comment:
Why not set `self.MPI_LAUNCHER` and `self.MPI_N` in CMake too ?

Reply:
Was done with `sys.platform` code untouched.
I don't think cmake can give us the mpi executable